libs/utils/logWebVitals.js

```diff
@@ -76,18 +76,22 @@ function isFragmentFromMep(fragPath, mep) {
   });
 }
 
+const boolStr = (val) => (val ? 'true' : 'false');
+
 function observeLCP(lanaData, delay, mep) {
+  const sectionOne = document.querySelector('main > div');
   new PerformanceObserver((list) => {
     const entries = list.getEntries();
     const lastEntry = entries[entries.length - 1]; // Use the latest LCP candidate
     lanaData.lcp = parseInt(lastEntry.startTime, 10);
     const lcpEl = lastEntry.element;
     lanaData.lcpElType = lcpEl.nodeName.toLowerCase();
     lanaData.lcpEl = getElementInfo(lcpEl);
+    lanaData.lcpSectionOne = boolStr(sectionOne.contains(lcpEl));
```

Comment:
I was wondering whether an empty document translates to a childless `main`, but it seems a `div` is always added inside. I was going to suggest some optional chaining, but it's not needed.